docs/getting-started/first-source.md

```diff
@@ -52,14 +76,15 @@ We can use the CloudEvents Player to send and receive CloudEvents. If you open t
 
     For more information on the CloudEvents Specification, check out the [CloudEvents Spec](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md){target=_blank}.
 
-Fill out the form with whatever you data you would like to (make sure your Event Source does not contain any spaces) and hit the "SEND EVENT" button.
+Fill out the form with whatever you data you would like to **(make sure your Event Source does not contain any spaces)** and hit the "SEND EVENT" button.
```

Comment:
1. Fill in the form with whatever you data you want.
1. Ensure your Event Source does not contain any spaces.
1. Click **SEND EVENT**.

Reply:
This is an especially great change, thank you